core/lib/types/src/abi.rs

```diff
@@ -365,3 +349,164 @@ impl Transaction {
         })
     }
 }
+
+pub struct ForceDeployment {
+    pub bytecode_hash: H256,
+    pub new_address: Address,
+    pub call_constructor: bool,
+    pub value: U256,
+    pub input: Vec<u8>,
+}
+
+impl ForceDeployment {
+    /// ABI schema of the `ProposedUpgrade`.
```

Comment:
nit: 
```suggestion
    /// ABI schema of the `ForceDeployment`.
```
same for methods below